semeio/jobs/design_kw/design_kw.py

```diff
@@ -83,11 +83,22 @@ def extract_key_value(parameters):
     # If key already exists, raise Exception
 
     res = {}
+    unparsable_lines = []
     for line in parameters:
-        key, value = line.split()
+        try:
+            key, value = line.split()
+        except ValueError:
+            unparsable_lines += [line]
+            continue
         if key in res:
             raise SystemExit(
```

Comment:
I think this should also be included in your list of unparsable_items, and instead of just adding the line to the list, you could add the whole error message, that makes it easier to expand it later should we want to add more validation. Then you can just loop over that list in the end if it exists before raising.

Reply:
Done